hawkular-datamining-rest/src/main/java/org/hawkular/datamining/rest/exception/mapper/IllegalStateExceptionMapper.java

```diff
@@ -15,31 +15,20 @@
  * limitations under the License.
```

Comment:
This is funny, I created new file ``IllegalStateExceptionMapper.java`` and it assumes that it is renamed ``EngineConfiguration.java``